test/unit/org/apache/cassandra/auth/MutualTlsAuthenticatorTest.java

```diff
@@ -86,7 +90,7 @@ public static void setup()
     @After
     public void after() throws IOException, TimeoutException
     {
-        MBeanWrapper.instance.unregisterMBean("org.apache.cassandra.auth:type=IdentitiesCache");
+        MBeanWrapper.instance.unregisterMBean("org.apache.cassandra.auth:type=IdentitiesCache", MBeanWrapper.OnException.IGNORE);
```

Comment:
I was wondering why this was needed, is it because `testValidMaxCertificateAgeConfiguration` unregisters after each pair, so at the end of the test the bean is no longer registered?

Reply:
yeah, this prevents the `after` method to fail because the bean is not registered